Credify/Credify/Views/LoadingView/LoadingView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  LoadingView.swift
+//  Credify
+//
+//  Created by Gioan Le
+//
+
+import UIKit
+import Lottie
+import SwiftUI
+
+class LoadingView: UIView {
+    private static let view: AnimationView = {
+        let v = AnimationView(name: "credify-loading", bundle: .serviceX)
```

Comment:
@lqgioan Is this the new loading animation? Please use the same file that the passport web app uses.

Reply:
@ngo275 sure, it's the same file that the passport web app uses